source/io/HttpClient.cpp

```diff
@@ -39,21 +39,49 @@ extern "C" {
 }
 #endif
 
+void AddCallChainToSourceIfErrorPresent(ErrorCluster *errorCluster, ConstCStr methodName)
```

Comment:
Would be helpful to add a comment showing some examples:

ie
If existing source is empty string then output is
"Primitive in MyVI->SubVI"

If existing source contains contents such as "<APPEND>\nCustom Error Information" then output is
"Primitive in MyVI->SubVI<APPEND>\nCustom Error Information"